package chapters.chapter12.CheckPoints12;

public class Checkpoints12_13 {

    /*
Suppose that statement2 causes an exception in the following try-catch block:
try {
  statement1;
  statement2;
  statement3;
}
catch (Exception1 ex1) {
}
catch (Exception2 ex2) {
}

statement4;


Answer the following questions:
1. Will statement3 be executed?  --> no
2. If the exception is not caught, will statement4 be executed?  --->no
3. If the exception is caught in the catch block, will statement4 be executed?  -->yes

     */
}
